vms/registration/views.py

```diff
@@ -164,12 +185,28 @@ def post(self, request):
                                     'volunteer_form': volunteer_form,
                                     'registered': self.registered,
                                     'phone_error': self.phone_error,
+                                    'password_error':self.password_error,
                                     'organization_list':
                                     self.organization_list,
                                 })
 
-                    user = user_form.save()
+                    vol_password=request.POST.get('usr-password')
```

Comment:
DITTO.
Try to include this with phone no. validation.

Reply:
@abhishekarya286 phone no.validation has already been done in a separate file phone_validate.py

Reply:
I mean to say check for password validation in the same if block.

Reply:
@abhishekarya286  Sir !, the 2 if blocks need to be independent of each other . If I go  on to include the password validation in the phone_validation of block, password_validation error will only be thrown if there's been a phone_validation error.

Reply:
You just need to put in a small logic there and it can be done then. Just try it out. If it doesn't work out then no problem but I would really suggest you to try.